controllers/api/searchRoutes.js

```diff
@@ -0,0 +1,109 @@
+const router = require("express").Router();
+
+// Get call for team info
+router.post("/", async (req, res) => {
+  const teamSearchUrl = req.body.url + req.query.term;
```

Comment:
Best practice to move this (and by extension the lines following) inside the try-catch in case the route is misused and the URL and term are undefined. Still satisfies MVP however.